### ISSUE TITLE
Alternative for using the apkbuilder command line tool

### ISSUE THREAD

**Issue by hrj** [jberkel/android-plugin#23]
When I execute "package-debug", the final step shows a message (logged as error, but is actually just a warning):

```
[debug] Executing command [//android-sdk-linux_86/tools/apkbuilder, /tmp/android_scala/target/scala_2.8.1/dsc_test_2.8.1-1.0.apk, -z, /tmp/android_scala/target/scala_2.8.1/resources.apk, -f, /tmp/android_scala/target/scala_2.8.1/classes.dex]
[error] 
[error] THIS TOOL IS DEPRECATED. See --help for more information.
[error] 
[info] == package-debug ==
[success] Successful.
[info] 
```

When I run apkbuilder from the command line, this message is shown:
    <<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<
    THIS TOOL IS DEPRECATED and may stop working at any time!

```
If you wish to use apkbuilder for a custom build system, please look at the
com.android.sdklib.build.ApkBuilder which provides support for
recent build improvements including library projects.
>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
```

**Comment by jberkel** [comment]
see https://github.com/jayway/maven-android-plugin/pull/6 for a possible solution

**Comment by paulbutcher** [comment]
Serendipity is a wonderful thing - I've just been looking at this, and was planning to take exactly the same approach as in the Maven plugin. I'll put a pull request together when I have something I'm happy with (unless anyone else already has something?)

**Comment by jberkel** [comment]
have only done a bit of research so far, pull request welcome :)

**Comment by jberkel** [comment]
closed in jberkel/android-plugin@28bae04
